examples/noisy_simulation_example.py

```diff
@@ -0,0 +1,43 @@
+"""
+Creates and simulate a noisy circuit using cirq.ConstantQubitNoiseModel class.
+"""
+import cirq
+
+
+def noisy_circuit_demo(amplitude_damp):
+    """Demonstrates a noisy circuit simulation.
+    """
+    q = cirq.NamedQubit('q')
+    noisy_circuit = cirq.Circuit.from_ops(
+        cirq.measure(q, key='initial_state'),
+        cirq.X(q),
+        cirq.measure(q, key='after_not_gate'),
+    )
+    results = cirq.sample(program=noisy_circuit,
+                          noise=cirq.ConstantQubitNoiseModel(
+                              cirq.amplitude_damp(amplitude_damp)),
+                          repetitions=100)
+    print("Noise model: ConstantQubitNoiseModel with",
```

Comment:
make this more decriptive like
"ConstantQubitNoiseModel with amplitude damping of rate"